### ISSUE TITLE
The specific route or code location cannot be located through the error information.

### ISSUE THREAD

**Issue by lwq057** [uNetworking/uWebSockets.js#1142]
The error message cannot locate the specific route or code location.

In the production environment, a large number of errors are found in the error log:

`Error: Returning from a request handler without responding or attaching an abort handler is forbidden!`

Because there are a lot of codes, I read and checked again, but still couldn't find the specific error location, which is very painful.

Can you show which route the error is, or give the number of code lines?

**Comment by uNetworkingAB** [comment]
> Can you show which route the error is

Yes that could be a good improvement. The URL could be displayed

**Comment by uNetworkingAB** [comment]
Displaying the URL (or maybe the route pattern) is a simple fix to this issue and I can see how it would be an issue in JS while in C++ you could use a debugger to find out where it broke.

This is probably the biggest ergonomic win of all issues right now

**Comment by uNetworkingAB** [comment]
This is fixed in recent version
